File: examples-sdk/go/804-trigger-cdc-postgres/main.go
// 804-trigger-cdc-postgres — Go SDK equivalent
// Compile: vil compile --from go --input 804-trigger-cdc-postgres/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("app", 8080)
	s.Compile()
}

File: examples-sdk/zig/804-trigger-cdc-postgres/main.zig
// 804-trigger-cdc-postgres — Zig SDK equivalent
// Compile: vil compile --from zig --input 804-trigger-cdc-postgres/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("app", 8080);
    server.compile();
}
